### PR TITLE
Archive and delete note

### PR DESCRIPTION
**Pull request by SanketDhabarde**
### Changes made
- Delete the note
- Add it inside the trash
- archive the note
- unarchive the note

### Preview link
[Preview](https://deploy-preview-11--always-notes.netlify.app/)

### CODE REVIEW

File: src/components/Note/Note.jsx [hunk 1]
```diff
@@ -2,31 +2,110 @@ import React from "react";
...
+            </div>
+          ) : (
+            <div
+              className="note-icon center-div"
+              title="Pin note"
+              onClick={pinNoteHandler}
+            >
+              <i className="fas fa-thumbtack unpin-note-icon"></i>
+            </div>
+          ))}
```
Comment (Naveenchandar) [inline src/components/Note/Note.jsx]: title of div and classname of icon is only getting changed right ?
Can you do ternary condition within title attribute of div and classname of icon ?
Why ternary condition for entire div element ?

File: src/components/Note/Note.jsx [hunk 2]
```diff
@@ -2,31 +2,110 @@ import React from "react";
...
+                </div>
+              ) : (
+                <div
+                  className="note-icon center-div"
+                  title="Archive"
+                  onClick={() => archiveNoteHandler("archive")}
+                >
+                  <i className="fas fa-archive"></i>
+                </div>
+              )}
```
Comment (Naveenchandar) [inline src/components/Note/Note.jsx]: Suggestion:
Please do ternary condition for title attribute instead of doing for entire element.

**Review by Naveenchandar — APPROVED** [review]
Given few suggestions. Please ignore if not needed.
Good folder structure.
File name consistency maintained.

File: src/components/Note/Note.jsx (hunk 1 above)
Comment (SanketDhabarde) [reply src/components/Note/Note.jsx]: Make sense. thank you so much for suggestion

File: src/App.js [hunk 3]
```diff
@@ -11,6 +11,7 @@ function App() {
       <Routes>
         <Route path="/" element={<Landing />} />
         <Route path="/home" element={<Home />} />
+        <Route path="/trash" element={<Trash />} />
       </Routes>
     </div>
```
Comment (aniketsingh13) [inline src/App.js]: suggestion,you can make folder for routes

File: src/components/Note/Note.jsx (hunk 2 above)
Comment (SanketDhabarde) [reply src/components/Note/Note.jsx]: Thank you for the suggestion. i will make the changes.

**Review by aniketsingh13 — APPROVED** [review]
folder structure is good, Ui also look clean

File: src/App.js (hunk 3 above)
Comment (SanketDhabarde) [reply src/App.js]: Sorry. I did not understand. folder for routes means? Do you mean a component where all routes will be present with `<Route/>` tags and use that component here?

Comment (aniketsingh13) [reply src/App.js]: yes

Comment (SanketDhabarde) [reply src/App.js]: Nice idea. I will do it if time permits. thanks for the suggestion.

**Pull request merged**